test/expected/sql_query_results_optimized.out

```diff
@@ -123,10 +123,9 @@ EXPLAIN (costs off) SELECT * FROM hyper_timefunc ORDER BY unix_to_timestamp("tim
 -----------------------------------------------------------------------------------------
```

Comment:
Note to self: This whole file needs to be re-checked after sort transform optimization is fixed.